R/validate.R

```diff
@@ -83,48 +132,61 @@ vt_validated_build <- function(pkg = ".",...) {
           class = "vt.buildFail")
   })
 
+  return(build_path)
+
 }
 
-#' Validate an installed package
+
+
+#' Validate and install package and Build package
 #'
-#' Rerun the validation report that is contained within a package that was built
-#' by `valtools::vt_validated_built()`. Saves the new report in the current directory.
+#' @importFrom rlang inform
+#' @importFrom utils install.packages
+#' @export
 #'
-#' @param pkg Name of installed package
-#' @param output_directory Location of directory to output validation report
-#' @param open open the validation report after the rendering is completed
+#' @rdname validate
 #'
-#' @returns Returns the path of the regenerated validation report
+vt_validate_install <- function(pkg = ".", src = pkg, ..., install_verbose = TRUE){
+  bundle <- vt_validate_build(pkg = ".", src = src, ...)
+  on.exit({unlink(bundle)})
+  install.packages(bundle, type = "source", repos = NULL, verbose = install_verbose, quiet = !install_verbose)
```

Comment:
What do you think about adding `INSTALL_opts = 'install-tests'` here? in case a part of the validation is to test the package tests.

Reply:
I think we can add that as a bool install_tests = TRUE/FALSE, and add that arg or not. would we want that as part of the build too?